### PR TITLE
Move sanitization logic to Go served by WASM

### LINKED ISSUE #19: Serve the Go code via WASM code to execute in the front end

### PR DESCRIPTION
**Pull request by the-c0d3br34k3r**
## Description
Closes #19.

This change ensures that the sanitization is done in the front end browser and also ensures that we can re-use the same code else where in the future, Eg: CLI tool, library etc.

## Author's Checklist
- [x] **These changes don't break existing dependants.** If they do, and you suggest to proceed, 
please explain why.

## Reviewer's Checklist
- [ ] **All classes have documentation comments.** -NA
- [ ] **All methods have documentation comments.** - NA
- [x] **No parameters are hardcoded.** Should be loaded as properties instead. If there needs to be hardcoded properties, explain why.
- [ ] **All information to be logged/displayed to the user are internationalized.** - Will be addressed in #21 
- [x] **README.md has been updated if needed.**
- [ ] **pom.xml version is set to the latest date** If not, explain why. - NA

## Assignee's Checklist
- [ ] **All GitHub action checks have passed.**
- [ ] **All reviewers have approved.**
- [ ] **Relevant documentation has been updated if needed.**
- [ ] **pom.xml version is set to the latest date** If not, explain why.

### CODE REVIEW

**Comment by the-c0d3br34k3r** [comment]
@silvs110 
- Ignore the ESLint actions failure, it needs to be removed. Not relevant to our project.
- The utility methods are in wasm_main.go. We will move it out into a separate file if needed in the future.
- For simple, self explanatory methods, I haven't added comments.
- Unit tests & End to end tests will be added in #9
- Logger mechanism will be added in #21

File: script/lib/js-yaml.min.js [hunk 1]
```diff
@@ -1,2 +0,0 @@
-/*! js-yaml 4.1.0 https://github.com/nodeca/js-yaml @license MIT */
```
Comment (silvs110) [inline script/lib/js-yaml.min.js:1]: Do we need to keep lib directory if it's empty?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,20 @@
...
+## Support
+The following browsers are supported:
+ - Google Chrome
+ - Firefox
+
 At the moment, it sanitizes HAR files.
 Support for sanitizing SAML requests/responses are in progress.
 
+## Build
+To build the WASM script, run the `build_was` script.
```
Comment (silvs110) [inline README.md:13]: I'm getting an error when only running this command:
```
# sanitizer
./sanitize.go:55:27: undefined: jsonpath.GetWithPaths
```
Are there any prerequisites to build the WASM script?

**Review by silvs110 — CHANGES_REQUESTED** [review]

File: script/lib/js-yaml.min.js (hunk 1 above)
Comment (the-c0d3br34k3r) [reply script/lib/js-yaml.min.js:1]: It's not empty. It has [other files as well](https://github.com/padaiyal/sanitizer/tree/issue19/script/lib).

File: README.md (hunk 2 above)
Comment (the-c0d3br34k3r) [reply README.md:13]: A version of `jsonpath` with `GetWithPaths` hasn't been released yet.

Use the `master` branch tip for installing `github.com/PaesslerAG/jsonpath`.
Something like:
```
go get github.com/PaesslerAG/jsonpath@master
```

The generated `index.html` and WASM code are stand alone and should work for you irrespective of the presence of the right dependency in your Go development environment.

**Review by silvs110 — APPROVED** [review]

**Pull request merged**